Assignment/FSMTest.c

```diff
@@ -0,0 +1,92 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Vinitha Perumal - GET
+// ------------------------------------------------------------------
+// Test.c
+// Program on A6B branch.
+// ------------------------------------------------------------------------------------------------
+#define _CRT_SECURE_NO_WARNINGS 1

```

Comment:
Build error, please check the project builds first
![err1](https://github.com/user-attachments/assets/cbeb5d24-4ce4-4c62-9c90-327452bafd06)